modules/react/collection/spec/useListItemAllowChildStrings.spec.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import {renderHook} from '@testing-library/react-hooks';
+import {useListModel, useListItemAllowChildStrings} from '@workday/canvas-kit-react/collection';
+
+describe('useListItemAllowChildSiblings', () => {
```

Comment:
```suggestion
describe('useListItemAllowChildStrings
```suggestion
describe('useListItemAllowChildSiblings', () => {
```', () => {
```